packages/help-text/README.md

```diff
@@ -0,0 +1,328 @@
+
+```html
+<sp-help-text variant="negative" icon>
+    Create a password with at least 8 characters.
+</sp-help-text>
+```
+
+## Disabled
+
+When associated to content the is disabled, use the `disabled` attribute to match the delivery of the `<sp-help-text>` element to that content.
+
+```html
+<sp-help-text disabled>Passwords must be at least 8 characters.</sp-help-text>
+```
+
+## helpTextManager
+
+The `createHelpTextManager()` helper method surfaces a `render()` method and `id` property to be leveraged within elements into which you'd like to be able to slot `<sp-help-text>` elements. The `id` property should be used while in `{ mode: 'internal' }` as part of the `aria-describedby` attribute of the element you would like to describe with your `<sp-help-text>` element(s). The `render(invalid?: boolean)` method accepts the optional `invalid` argument to manage the presence of the `neutral-text` and `negative-text` slots. These slots are stacked within the `help-text` slot, so that content addressed for that slot will always override content addressed to the other two, regardless of their presence.
+
+The `createHelpTextManager()` is called with default options of `{ mode: 'internal' }` and should be leveraged to associate slotted `<sp-help-text>` elements with content within the shadow root of the parent element.
+
+```js
+import { SpectrumElement, html } from '@spectrum-web-components/base';
+import { createHelpTextManager } from '@spectrum-web-components/help-text/manager.js';
+
+export class ParentWithDescribedElement extends SpectrumElement {
+    helpTextManager = createHelpTextManager({ mode: 'internal' });
+
+    invalid = false;
+
+    render() {
+        return html`
+            <input aria-describedby=${this.helpTextManager.id} />
+            ${this.helpTextManager.render(this.invalid)}
+        `;
+    }
+}
+```
+
+If you would like to describe the element into which you are slotting the `<sp-help-text>` element, use the `{ mode: 'external' }` option so that appropriate `id` and `aria-describedby` attributes will be applied by the helper.
+
+```js
+import { SpectrumElement, html } from '@spectrum-web-components/base';
+import { createHelpTextManager } from '@spectrum-web-components/help-text/manager.js';
+
+export class DescribedElement extends SpectrumElement {
+    helpTextManager = createHelpTextManager({ mode: 'external' });
+
+    invalid = false;
+
+    render() {
+        return html`
+            ${this.helpTextManager.render(this.invalid)}
+        `;
+    }
+}
+```
+
+### Usage with self managed validity
+
+When the parent element into which you are placing an `<sp-help-text>` manages its own validity in some way, you can leverage content in the `neutral-text` and the `negative-text` slots together and rely on the `helpTextManager()` supplied templating to surface the appropriate one based off of that internal state.
```

Comment:
I was trying to figure out of the `HelpTextManager` listened for changes in its parent's `.invalid` state or something - I think the reality is less general than currently described here. This won't automatically work on any component that manages its own validity in some way... that component needs to specifically be designed to interact with `<sp-help-text>`:

```suggestion
Spectrum Web Components that support automatic validation, like `<sp-fieldgroup>` and `<sp-textfield>`, automatically render either `neutral-text` or `negative-text` slots based on validity. Provide both, and the appropriate `<sp-help-text>` element will be surfaced:
```